src/black/__init__.py

```diff
@@ -573,7 +621,11 @@ def main(
 
     if verbose or not quiet:
         out("Oh no! 💥 💔 💥" if report.return_code else "All done! ✨ 🍰 ✨")
-        click.secho(str(report), err=True)
+        if isinstance(report, JunitReport):
+            junitxml.write(str(report))  # type: ignore
```

Comment:
Why do you need an ignore?

Reply:
I had to add ignore because:

junitxml is from type ```click.utils.LazyFile``` and ```mypy``` says that ```LazyFile``` has no attribute ```write```.